proposals/62-nft-storage-for-nft-hack.md

```diff
@@ -0,0 +1,184 @@
+
+    * Implement a [deal batching service](https://github.com/protocol/web3-dev-team/pull/60) prototype for increased successful deal probability.
+
+    * Run a "[Free Retrieval via IPFS](https://github.com/protocol/web3-dev-team/pull/52) Lotus node" that `nft.storage` can make deals with and that will expose the NFT data to IPFS.
+
+    * Build and run a "[Retrieval from Filecoin](https://github.com/protocol/web3-dev-team/pull/57) IPFS node" that will allow NFT data we store in deals to be pulled directly from Filecoin and available via IPFS.
+
+* Create a single page website at `nft.storage` explaining how to use the service with registration for API keys.
+
+* Implement the [remote pinning service in js-ipfs](https://github.com/protocol/web3-dev-team/pull/58) so that it can be used in web based NFT hacks.
+
+    If this is not ready on time, folks can always use the Pinata API directly. Obviously, for non-web based hacks users can run a go-ipfs node, configured to use `nft.storage` as their remote pinning service.
+
+Essentially, everything after building a simple pinning service API and website is a bonus and won't effect QoS for NFTHack participants negatively.
+
+It gives us purpose, a (albeit soft) deadline and a safe area for building out and deploying project proposals _in production_ that directly address PMF issues like deal flow and Filecoin ↔️ IPFS interop.
+
+As an added bonus, we'll be dogfooding our own tech and ideas for greater understanding and appreciation of any difficulties. It also will help to validate assumptions made in our project proposals.
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve. Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives. 
+-->
+
+A remote pinning service API and a simple website for information and registration will exist for use by NFTHack participants to store NFT data for free for a limited time.
+
+This will cement IPFS as the primary means of off-chain NFT storage and will raise awareness of Filecoin as a storage provider.
+
+Currently a lot of NFT data is stored on IPFS with no clear story for permenance. Leveraging the remote pinning service API and providing guarantees of availability for the duration of the hack will prompt developers building NFT related software to think about permenance and who provides it. It will present Filecoin as an option for consideration and will likely drive some traffic to pinning services like Pinata.
+
+#### Assumptions &amp; hypotheses
+_What must be true for this project to matter?_
+<!--(bullet list)-->
+
+* NFT users want to store NFT data on IPFS.
+* NFT users want guarantees of data availability.
```

Comment:
I would say one others is that NFT users (though perhaps these should be called platforms to not overload a term) care about their end users (Artists) being able to take control of their data.

I think for an MVP we could define this as "An artist would be able to retrieve their data even if X app went away tomorrow"